### ISSUE TITLE
Missing CPU: Lists 7 of 8 expected

### ISSUE THREAD

**Issue by xxxserxxx** [xxxserxxx/gotop#19]
htop shows activity on all 8 CPUs (i7 8550u)

Original https://github.com/cjbassi/gotop/issues/108
Submitter @dkrieger, @syslino

**Comment by xxxserxxx** [comment]
Is the label not showing for the 8th CPU? If so, can you check that it's not being cropped due to the window size? Can you post a screenshot? Thanks!

**Comment by xxxserxxx** [comment]
![gotop](https://user-images.githubusercontent.com/19599607/52529965-936e3500-2cca-11e9-8e94-737e18d4c630.png)

using gnome-terminal on gnome 3.30.2 (wayland) @ 1440p w/ 1.2 font scaling (tested after screenshot w/ no font scaling, same result). Worth noting that running gotop in a VM running sway on arch (using termite) on this machine shows 8 cpus, and that was with the display set to 1080p.

**Comment by xxxserxxx** [comment]
... after trying it in a different terminal and yanking that region when running within vim, I've confirmed the 8th CPU is displayed w/ the background color, making it invisible. I'm using the bog-standard gnome-terminal solarized colorscheme. Perhaps a more conservative foreground colorscheme option would help? I noticed (less extreme) readability issues in the VM as well.

Edit: On a related note, it looks great when specifying the solarized colorscheme! From a usability perspective, I guess I'd recommend limiting the default color palette, leaving it to ricers to create an alias w/ their preferred "rich" colorscheme.

**Comment by xxxserxxx** [comment]
That's interesting. So the default gotop colorscheme only uses like the first 8 basic xterm colors, but it looks like your terminal solarized colorscheme is modifying those to look different. The 8th cpu line is xterm color 8 which is grey but it looks like its being changed into the solarized background color. Also, the swap line is supposed to be yellow but it looks grey in the screenshot, stuff like that.

So should we change anything in gotop? If gotop's solarized colorscheme is working for you then I would say just use that :D

**Comment by xxxserxxx** [comment]
Hi, I've got a very similar issue. I've got an Intel i5-5200U which _should_ report 4 CPU's – and does so in any other monitor application. With gotop Im getting only three CPU's shown.
I'm on Arch Linux, using termite. I tried to yank the output in a nvim terminal-buffer but still – three CPU's shown (index starting at 0). 

Edit: No colorscheme does work for me..

**Comment by xxxserxxx** [comment]
@dkrieger, @syslino, at some point in my journeys through Caleb's code I noticed in issue with color assignment where if the algorithm ran out of colors it used the background color.  This was fixed in the past week or so and is available on the master branch, and in v4.0.

If either of you are still using gotop and have the ability to pull, build, and check this, I'd appreciate the confirmation that it's addressed.

Thank you
